src/v3/src/transformer/selectAuthenticator/utils.test.ts

```diff
@@ -191,6 +191,7 @@ describe('Select Authenticator Utility Tests', () => {
         expect(currentOption?.label).toBe(option.label);
         expect(currentOption?.options.ctaLabel)
           .toBe('oie.verify.authenticator.button.text');
+        // @ts-expect-error OKTA-554038
```

Comment:
Is a different ticket needed for these type issues? Or can the ticket be amended with these props?

Reply:
Good question. I'll check with SDK team

Reply:
Updated to OKTA-555811. Thakns